src/kafka-consumer.h

```diff
@@ -104,6 +107,7 @@ class KafkaConsumer : public Connection {
   bool m_is_subscribed = false;
 
   void* m_consume_loop = nullptr;
+  NodeKafka::Callbacks::QueueNotEmpty m_queue_not_empty_cb;
```

Comment:
This is in the same namespace, can be avoided
```suggestion
  Callbacks::QueueNotEmpty m_queue_not_empty_cb;
```